test/test_lazy.py

```diff
@@ -274,6 +290,107 @@ def lazy_to_eager(u):
     assert is_close, f"{lhs} not <= {rhs}"
 
 
+def _get_poiseuille():
```

Comment:
Could this be done by parametrizing the 'normal' Navier-Stokes tests? (What's preventing that? Any way to avoid the blob'o'copy-paste?)

Reply:
I guess the intent of these tests is to compare lazy to eager, rather than lazy to exact. I suspect we can use tighter tolerances here than we can in the regular NS tests. Whether this is actually useful... I'm not sure. @kaushikcfd Any thoughts?

Reply:
If the 'normal' NS tests take a long time to execute (for reasons like large problem size), it might be better to check the equivalence of the lazy v eager operator evaluations for a smaller problem as the PytatoArrayContextForTests results in a single-threaded operator application. Otherwise, parametrizing seems like the cleaner approach.